spec/relativetimeformat.html

```diff
@@ -30,8 +30,8 @@ <h1>InitializeRelativeTimeFormat (relativeTimeFormat, locales, options)</h1>
         1. Let _dataLocale_ be _r_.[[DataLocale]].
         1. Let _s_ be ? GetOption(_options_, `"style"`, `"string"`, «`"long"`, `"short"`, `"narrow"`», `"long"`).
         1. Set _relativeTimeFormat_.[[Style]] to _s_.
-        1. Let _t_ be ? GetOption(_options_, `"type"`, `"string"`, «`"numeric"`, `"text"`», `"numeric"`).
-        1. Set _relativeTimeFormat_.[[Type]] to _t_.
+        1. Let _numeric_ be ? GetOption(_options_, `"numeric"`, `"string"`, «`"always"`, `"auto"`», `"always"`).
```

Comment:
On https://github.com/tc39/proposal-intl-relative-time/issues/54#issuecomment-360378693, there's a mention about changing the default to `auto`. Are we going to change or keep `always` numeric?

I also prefer `auto`, because it has a better output for the case listed here: https://github.com/tc39/proposal-intl-relative-time/issues/58.

Reply:
Given the two options `always` and `auto` I would expect `auto` to be the default regardless of what it is... 😄

Reply:
+1 for `auto` by default

Reply:
I don't think we should go for `auto`by default. I raised this issue here: https://github.com/tc39/proposal-intl-relative-time/issues/9#issuecomment-347376066

Basically, the textual representation ("yesterday", "tomorrow" etc.) is potentially misleading if you do not use an algorithm to compute time distance relative to the anchor in time.

For that reason, I suggest that the default value be the one that works irrelevant of calculations: "1 day ago" works for any 24h period, irrelevant of if it's quarter past midnight or before.

Reply:
I agree with @zbraniecki here! this will be a very low level API, mostly used by other libraries to do the right computation for you, in which case having "auto" should be an opt-in mechanism.